.buildkite/x-pack/pipeline.xpack.heartbeat.yml

```diff
@@ -25,39 +25,189 @@ env:
   #Deps
   ASDF_MAGE_VERSION: 1.15.0
 
+notify:
+  - github_commit_status:
+      context: "$BEATS_PROJECT_NAME"
```

Comment:
shall we remove this, since we've decided to have notifications per step (which already exist in this PR)?